### PR TITLE
Feat/sns voting result

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Display voting results of sns proposal.

# Changes

- refactor `VotesResults` to get number values
- use `VotesResults` in `SnsProposalVotingSection`

# Tests

- SnsProposalVotingSection should display voting results component

# Screenshots

image

### CODE REVIEW

File: frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts
Comment (lmuntaner) [inline frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts:1]: As another PR, could you add also a page-object for this section?

There are some examples in the project already and me or @dskloetd can help you understand them

File: frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  it("should render vote results", () => {
+    const { queryByText } = render(SnsProposalVotingSection, {
+      props: {
+        proposal: {
+          ...proposal,
+        },
+      },
+    });
+
+    expect(queryByText(en.proposal_detail.voting_results)).toBeInTheDocument();
```
Comment (lmuntaner) [inline frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts:25]: Ideally we could expect that the proper `yes` and `no` are displayed.

Now that you pass the proposal and the logic of splitting into yes and no are here it makes sense.

File: frontend/src/tests/lib/components/proposal-detail/VotesResults.spec.ts [hunk 2]
```diff
@@ -14,15 +13,16 @@ describe("VotesResults", () => {
   let renderResult: RenderResult<SvelteComponent>;
   let yes: number, no: number;
   beforeEach(() => {
-    neuronsStore.setNeurons({ neurons: [], certified: true });
+    yes = Number(mockProposalInfo.latestTally?.yes) / E8S_PER_ICP;
```
Comment (lmuntaner) [inline frontend/src/tests/lib/components/proposal-detail/VotesResults.spec.ts:16]: I wouldn't create this from a proposalInfo. I would just pass the numbers and check that the passed numbers are rendered.

**Review by lmuntaner — COMMENTED** [review]
A few comments

File: frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts:1]: Added a todo in notion

File: frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/sns-proposals/SnsProposalVotingSection.spec.ts]: Good point, updated!

File: frontend/src/tests/lib/components/proposal-detail/VotesResults.spec.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/proposal-detail/VotesResults.spec.ts]: True, much better so.

**Review by peterpeterparker — APPROVED** [review]
LGTM

**Pull request merged**